ScoutSuite/providers/azure/rules/rulesets/default.json

```diff
@@ -206,6 +206,18 @@
                 "enabled": true,
                 "level": "warning"
             }
+        ],
+        "storageaccount-trusted-microsoft-services-enabled.json": [
```

Comment:
Looks like this rule is there twice?

Reply:
Ok this is now fixed. I think I mixed some code